CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.14.0]
+### Changed
+- **Breaking:** The minimum version of Node.js required for this package has been bumped to v14.
```

Comment:
Nit: To match formatting we use on other repos, and to add a newline between sections as is required by our auto-changelog linting elsewhere (not sure why the linter isn't setup here :thinking: )

```suggestion
- **BREAKING:** The minimum version of Node.js required for this package has been bumped to v14.

```